units/heroes/Brainless_Heroes.cfg

```diff
@@ -0,0 +1,222 @@
+#textdomain wesnoth-Danse_Macabre
+[unit_type]
+    id=Brainless Hero
+    [base_unit]
+        id=Brain Eater
+    [/base_unit]
+    advances_to=Battle Hero
+
+    # should be enabled in the end, since we
+    # a) don't want players in the beginning to see what heros will appear later
+    # b) nor do they have to see how the heroes look on higher levels (keeping some surprise)
+    # c) like the base_unit, they are shown in the help as advancements of the Soulless, which looks a bit odd
+    # d) the part without [variation] of this file is unused
+    # hide_help=yes
```

Comment:
If we add hide_help=yes here, then the help is not working as expected anymore.
What do you think about how it is right now?

It can be worked around with having the heroes in separate files, which use [base_unit] on the main unit, and hide all inherited variations in them. The help system will have for these heroes then a link to the base unit, but not showing the other heroes. But it comes with a lot of code.